src/mongo/db/storage/hse/SConscript

```diff
@@ -41,20 +41,6 @@ K_CON_SRC = Dir('.').srcnode().abspath
 K_CON_VERSION_FILE = os.path.join(K_CON_SRC, 'VERSION')
 K_GEN_FILE_NAME = os.path.join(K_CON_SRC, 'src', 'hse_versions.h')
 
-def getHseGitSha(conf, env):
```

Comment:
Do we need to change the markdown documents at all?

Reply:
There are some, updated.